postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -52,6 +52,13 @@ instances:
     #
     # ssl: false
 
+    ## @param timeout - integer - optional
+    ## Abort any statement that takes more than the specified number of milliseconds, starting from the time the command
+    ## arrives at the server from the client. A value of zero (the default) turns this off.
+    ## Cancelled queries won't log any metrics
+    #
+    # timeout: 1000
```

Comment:
I think we should name it query_timeout, as it may be nice to have a connect timeout as well.